test/.eslintrc

```diff
@@ -0,0 +1,2 @@
+rules:
+  import/no-extraneous-dependencies: [error, { devDependencies: true }]
```

Comment:
@hisabimbola we can fix this now. The glob feature was released in version `1.15.0` https://github.com/benmosher/eslint-plugin-import/blob/master/CHANGELOG.md.